src/gui/sdlmain.cpp

```diff
@@ -1168,12 +1187,13 @@ static void setup_presentation_mode(FRAME_MODE &previous_mode)
 		return;
 	previous_mode = mode;
 
-	// Configure the pacer. We only use it for graphical VFR modes because
-	// CFR modes determine if the frame is presented based on the
-	// scheduler's accuracy.
+	// Configure the pacer in VFR modes to detect when the host takes too
+	// long to render frames. CFR modes use the PIC scheduler to place
+	// frames and use their own measurements, so we disable the pacer for
+	// them.
 	const auto is_vfr_mode = mode == FRAME_MODE::VFR ||
 	                         mode == FRAME_MODE::THROTTLED_VFR;
-	render_pacer.SetTimeout(is_vfr_mode && !in_text_mode ? sdl.vsync.skip_us : 0);
+	render_pacer.SetTimeout(is_vfr_mode ? sdl.vsync.skip_us : 0);
```

Comment:
`us` stands for microseconds in `skip_us`? That's a bit unfortunate; I'd rather use `skip_micros`

Reply:
I'll rename these too. Thanks for the better suggestion!

There are some that are milliseconds (`_ms_`), I'll move these to millis.